_docs/_latest/setup/real-time-enforcer.md

```diff
@@ -0,0 +1,92 @@
+
+## Setting up Forseti Real Time Enforcer
+
+In your `main.tf` file, include the following Forseti Real Time Enforcer specific modules in the order provided:
+1. `real_time_enforcer_roles` creates and assigns the custom roles required for Real Time Enforcer to access and remediate resources.
+1. `real_time_enforcer_organization_sink` creates and sets up an organization 
+level [logging sink](https://cloud.google.com/logging/docs/api/tasks/exporting-logs) 
+and a [Pub/Sub topic](https://cloud.google.com/pubsub/docs/overview) to publish to.
+1. `real_time_enforcer` creates and sets up the necessary resources to run Forseti Real Time Enforcer that are not covered by the first two modules, which include:
+    * Real Time Enforcer VM running [Container Optimized OS](https://cloud.google.com/container-optimized-os/docs/concepts/features-and-benefits)
+    * Real Time Enforcer Service account
+    * GCS bucket with Real Time Enforcer policy files
+    * Pub/Sub subscription
+    * Forseti specific firewall rules
+    * Real Time Enforcer application Docker image
+    * Open Policy Agent server Docker image
+
+```
+module "forseti" {
+  source                   = "terraform-google-modules/forseti/google"
+  project_id               = "${var.project_id}"
+  gsuite_admin_email       = "${var.gsuite_admin_email}"
+  org_id                   = "${var.org_id}"
+  domain                   = "${var.domain}"
+  client_instance_metadata = "${var.instance_metadata}"
+  server_instance_metadata = "${var.instance_metadata}"
+}
+
+module "real_time_enforcer_roles" {
+   source = "terraform-google-modules/forseti/google/modules/real_time_enforcer_roles"
+   org_id = "${var.org_id}"
+   suffix = "${module.forseti.suffix}"
+}
+
+module "real_time_enforcer_organization_sink" {
+  source            = "terraform-google-modules/forseti/google/modules/real_time_enforcer_organization_sink"
+  pubsub_project_id = "${var.project_id}"
+  org_id            = "${var.org_id}"
+}
+
+ module "real_time_enforcer" {
+   source                     = "terraform-google-modules/forseti/google/modules/real_time_enforcer"
+   project_id                 = "${var.project_id}"
+   org_id                     = "${var.org_id}"
+   enforcer_instance_metadata = "${var.instance_metadata}"
+   topic                      = "${module.real_time_enforcer_organization_sink.topic}"
+   enforcer_viewer_role       = "${module.real_time_enforcer_roles.forseti-rt-enforcer-viewer-role-id}"
+   enforcer_writer_role       = "${module.real_time_enforcer_roles.forseti-rt-enforcer-writer-role-id}"
+   suffix                     = "${module.forseti.suffix}"
+}
+```
+
+## Run Terraform
+
+1.  Run `terraform init` to get the modules and plugins.
+    * If you have already installed Forseti using Terraform previously, you can skip step 3 and add the following 
+    roles to your `cloud-foundation-forseti` service account:
+        * **On the organization:**
+            * `roles/iam.organizationRoleAdmin`
```

Comment:
nit: do you think we can pull these in by `include`?

Reply:
Messes up the formatting, so just kept as is.